ref-exchange/src/account_deposit.rs

```diff
@@ -43,11 +91,23 @@ impl Account {
         self.tokens.insert(token.clone(), value - amount);
     }
 
+    /// Updates the account atorage usage,
+    /// Panic if there is not enough $NEAR to cover.
+    pub fn update_storage(&mut self, tx_start_storage: StorageUsage) {
+        // note: can not use += , 
+        // or a pannic with `attempt to subtract with overflow` emerges,
+        // if `tx_start_storage` is larger than `env::storage_usage()`
+        self.storage_used = self.storage_used + env::storage_usage() - tx_start_storage;
```

Comment:
this seems wrong.

self.storage_used is the current storage and env::storage_usage is global storage. Any other write or change will throw this function off. Or if there are two accounts been changed in one function.

generally, just need two methods:
 - `add_storage(bytes: StorageUsage)` - just adds to `self.storage_used`  and asserts
 - `remove_storage(bytes: StorageUsage)`  - just subtracts from `self.storage_used`

https://github.com/ref-finance/ref-contracts/blob/ref-components/ref-components/src/account_impl.rs#L26

and it's used like this https://github.com/ref-finance/ref-contracts/blob/ref-components/ref-escrow/src/lib.rs#L204